Sources/Rendering/OpenGL/VolumeMapper/index.js

```diff
@@ -779,6 +822,40 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
     }
   };
 
+  publicAPI.getClippingPlaneShaderParameters = (cellBO, ren, actor) => {
+    if (model.renderable.getClippingPlanes().length > 0) {
+      const keyMats = model.openGLCamera.getKeyMatrices(ren);
+
+      const clipPlaneNormals = [];
+      const clipPlaneDistances = [];
+
+      const clipPlanes = model.renderable.getClippingPlanes();
+      const clipPlaneSize = clipPlanes.length;
+      for (let i = 0; i < clipPlaneSize; ++i) {
+        const clipPlaneNormal = clipPlanes[i].getNormal();
+        const clipPlanePos = clipPlanes[i].getOrigin();
+
+        vec3.transformMat3(
+          clipPlaneNormal,
+          clipPlaneNormal,
+          keyMats.normalMatrix
+        );
+
+        vec3.transformMat4(clipPlanePos, clipPlanePos, model.modelToView);
+
```

Comment:
I believe here model.modelToView should instead be keyMats.wcvc as the clip planes are specified in world coordinates.

Reply:
It is correct to be transformed to world coordinates. I have reflected what you commented on.